alibi/prototypes/protoselect.py

```diff
@@ -617,10 +612,69 @@ def visualize_image_prototypes(summary: 'Explanation',
 
     # compute how many correct labeled instances each prototype covers
     idx, counts = np.unique(neigh_idx[y_protos[neigh_idx] == y_train], return_counts=True)
+    return {
+        'prototypes_indices': idx,
+        'prototypes_importance': counts,
+        'X_protos': X_protos[idx],
+        'X_protos_ft': X_protos_ft[idx]
+    }
+
+
+def visualize_image_prototypes(reducer: Callable[[np.ndarray], np.ndarray],
+                               summary: 'Explanation',
+                               trainset: Tuple[np.ndarray, np.ndarray],
+                               preprocess_fn: Optional[Callable[[np.ndarray], np.ndarray]] = None,
+                               knn_kw: Optional[dict] = None,
+                               ax: Optional[plt.Axes] = None,
+                               fig_kw: Optional[dict] = None,
+                               image_size: Tuple[int, int] = (28, 28),
+                               zoom_lb: float = 1.0,
+                               zoom_ub: float = 3.0) -> plt.Axes:
+    """
+    Plot the images of the prototypes at the location given by the `reducer` representation.
+    The size of each prototype is proportional to the logarithm of the number of assigned training instances correctly
+    classified according to the 1-KNN classifier (Bien and Tibshirani (2012): https://arxiv.org/abs/1202.5933).
+
+    Parameters
+    ----------
+    reducer
+        2D reducer. Reduces the input feature representation to 2D. Note that the reducer operates directly on the
+        input instances if ``preprocess_fn=None``. If the `preprocess_fn` is specified, the reducer will be called
+        on the feature representation obtained after passing the input instances through the `preprocess_fn`.
+    summary, trainset, preprocess_fn, knn_kw
+        See :py:meth:`alibi.prototypes.protoselect.compute_prototypes_importance` method.
```

Comment:
I would say this needs a full docstring as will be the main user-facing function. I believe the new `compute_...` function we would not advertise as much? In any case, if both are public, I would just duplicate the docstring.